tests/cmd/builds/commands.go

```diff
@@ -18,17 +18,22 @@ import (
 const ExampleImage = "deis/example-dockerfile-http"
 
 // Create executes `deis builds:create` as the specified user.
-func Create(user model.User, app model.App) {
-	createOrPull(user, app, "builds:create")
+func Create(user model.User, app model.App, opts ...string) {
+	createOrPull(user, app, "builds:create", opts)
 }
 
 // Pull executes the `deis pull` shortcut as the specified user.
-func Pull(user model.User, app model.App) {
-	createOrPull(user, app, "pull")
+func Pull(user model.User, app model.App, opts ...string) {
+	createOrPull(user, app, "pull", opts)
 }
 
-func createOrPull(user model.User, app model.App, command string) {
-	sess, err := cmd.Start("deis %s --app=%s %s", &user, command, app.Name, ExampleImage)
+func createOrPull(user model.User, app model.App, command string, opts []string) {
+	image := opts[0]
```

Comment:
I think a struct is overkill as well.  I have encountered many, many examples of variadic arguments being used in this fashion.  I'll concede that just because I've frequently seen it done does not implicitly make it right.  But I don't see the harm in it either.  This is all largely a matter of opinion.  Ultimately, I am fine with wherever we land on this _if_ the end result is that invocations of the method are clear and concise.  In a test suite like this, I would consider clarity and conciseness paramount.  Each spec tells a story and weird, extraneous details (like needing to create a struct that encapsulates some options) takes you out of that story.